BundID/Views/SupplementaryViews/DialogView.swift

```diff
@@ -8,12 +8,14 @@ struct DialogView<Action>: View {
     var imageMeta: ImageMeta?
     var secondaryButton: DialogButtons<Action>.ButtonConfiguration?
     var primaryButton: DialogButtons<Action>.ButtonConfiguration?
+    var infoBoxContent: InfoBoxContent?
```

Comment:
Hmm, should we move this property further up so it matches the structure of the screen layout? Or are we tying our code layout too much to the design? I think the matching initializer properties help imagining the screen (title, infobox, message, image, buttons).

Reply:
Good idea, did not think about this. I will update it before merging